### PR TITLE
Add Request Authentication UI

### PR DESCRIPTION
**Pull request by triffer**
**Description**

Changes proposed in this pull request:

- Add UI for Request Authentication.
- Add retries for UI tests to handle flaky tests.

**Related issues**

- https://github.com/kyma-project/istio/issues/581

### CODE REVIEW

**Review by kolodziejczak — CHANGES_REQUESTED** [review]
I assume that we are skipping RequestAuthentication's spec.targetRefs field, because at this moment we do not support k8s Gateway, is that correct?

File: config/ui-extensions/requestauthentications/details
[the remark this answers is not in the document]
Comment (triffer) [reply config/ui-extensions/requestauthentications/details:1]: Some time ago we decided to cover only basic scenarios in the UI. That's why I skipped targetRefs and some of the fields in jwtRule to keep it simple. Also it's easy to extend those fields if they are requested.

[the remark this answers is not in the document]
Comment (kolodziejczak) [reply config/ui-extensions/requestauthentications/details:1]: ok, thanks!

**Review by kolodziejczak — APPROVED** [review]

File: docs/release-notes/1.7.0.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## New Features
 
 - Allow for opting out of the **ENABLE_EXTERNAL_NAME_ALIAS** Istio pilot environment variable in the Istio custom resource. This allows for retaining behavior that was present in Istio prior to version 1.21. See issue [#787](https://github.com/kyma-project/istio/issues/787 ).
-- Update the Istio version to 1.21.2 [#802](https://github.com/kyma-project/istio/pull/802). Read [Istio 1.21.2 Release Announcement](https://istio.io/latest/news/releases/1.21.x/announcing-1.21.2/) and [Change Notes](https://istio.io/latest/news/releases/1.21.x/announcing-1.21/change-notes/) for more details.
\ No newline at end of file
+- Update the Istio version to 1.21.2 [#802](https://github.com/kyma-project/istio/pull/802). Read [Istio 1.21.2 Release Announcement](https://istio.io/latest/news/releases/1.21.x/announcing-1.21.2/) and [Change Notes](https://istio.io/latest/news/releases/1.21.x/announcing-1.21/change-notes/) for more details.
+- Add Request Authentication UI for Kyma Dashboard [#816](https://github.com/kyma-project/istio/pull/816)
```
Comment (nataliasitko) [inline docs/release-notes/1.7.0.md:5]:
```suggestion
- Add Request Authentication UI for Kyma dashboard [#816](https://github.com/kyma-project/istio/pull/816)
```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**